.devcontainer/Dev - Server Ffmpeg/install-ffmpeg.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+wget https://repo.jellyfin.org/releases/server/ubuntu/versions/jellyfin-ffmpeg/6.0.1-1/jellyfin-ffmpeg6_6.0.1-1-jammy_amd64.deb -O ffmpeg.deb
```

Comment:
It would probably be better to install the Jellyfin repo so this doesn't need to be updated for every ffmpeg update

Reply:
i woundered if it would be possible to just `apt instal ffmpeg6` or alike. 
is it sufficient to just follow: "https://jellyfin.org/docs/general/installation/linux#repository-manual" for thos until step 6?